src/main/webapp/WEB-INF/category.jsp

```diff
@@ -14,10 +14,20 @@
   <head>
     <title><%=countryCode%> <%=category%></title>
     <meta charset="UTF-8">
+    <meta name="viewport" content="initial-scale=1.0">
     <link rel="stylesheet" href="/css/main.css">
     <link rel="stylesheet" href="/css/user-page.css">
     <script src="/js/message-loader.js"></script>
+    <script src="/js/location.js"></script>
     <script src="https://cdn.ckeditor.com/ckeditor5/11.2.0/classic/ckeditor.js"></script>
+    <style>
+      #map {
+        height: 75px;
+      }
+      html, body {
```

Comment:
Please remove. You shouldn't need these styles applied to html & body.

Reply:
Apparently, if I don't use them the map can't be displayed

Reply:
Done